admin_manual/configuration/server/occ_command.rst

```diff
@@ -1413,6 +1414,18 @@ Remove a certificate:
 
  sudo -u www-data php occ security:remove [certificate name]
 
+.. _ransomeware_commands_label:
+
+Ransomeware
+-----------
+
+Use these commands to help protect your system against Ransomeware.
+
+::
+
+  occ ransomguard:scan <timestamp> <user>    Scan the ownCloud database for anomalies in a user account.
```

Comment:
I think ``ransomware:scan`` reports the file changes (new/mov/mod/del) since a certain time. So "anomalies" does not sound  correct here.

Reply:
Agree @phil-davis. The command reports all changes in the account starting from the specified time. It therefore allows the admin to "scan for anomalies".

Reply:
Commented in the conversation.